config/src/lib.rs

```diff
@@ -59,6 +59,10 @@ pub struct ConnectionsConfig {
     /// used as a bootstrap mechanism to gain access to the P2P
     /// network
     pub known_peers: Vec<SocketAddr>,
+
+    #[serde(default = "ConnectionsConfig::default_storage_peers_period")]
+    /// Period to dump peers into storage
+    pub storage_peers_period: u64,
```

Comment:
It is more idiomatic to store a time period using the [`Duration` type](https://doc.rust-lang.org/std/time/struct.Duration.html):
```suggestion
    pub storage_peers_period: Duration,
```

And then use it as:
```rust
let period = Duration::from_secs(30);
let seconds = period.as_secs();
```

Reply:
done!!